client/js/test/table-view.test.js

```diff
@@ -8,34 +8,109 @@ describe('table-view', () => {
         const fixturePath = './client/js/test/fixtures/sheet-container.html';
         const html = fs.readFileSync(fixturePath, 'utf8');
         document.documentElement.innerHTML = html;
-    })
+    });
+
+    describe('highlighting entire columns and rows', () => {
```

Comment:
These are good tests, but so far they test only for happy, expected paths. Add at least 2-3 others that test against edge cases. For instance, it('displays numeric total even when non-numbers are entered into column cells').